components/logins/ios/Logins/RustPasswordAPI.h

```diff
@@ -79,6 +79,10 @@ uint8_t sync15_passwords_delete(Sync15PasswordEngineHandle handle,
                                 char const *_Nonnull id,
                                 Sync15PasswordsError *_Nonnull error);
 
+uint8_t sync15_passwords_check_valid(Sync15PasswordEngineHandle handle,
```

Comment:
@thomcc Is this what I needed to add for swift?

Reply:
Part of it, but not all. This is basically the equivalent to the bit you did in PasswordSyncAdapter.kt. You also need to expose it in https://github.com/mozilla/application-services/blob/master/components/logins/ios/Logins/LoginsStorage.swift Let me know if you need any help doing so.

We also might want to expose information about how the record is invalid if they need that, @eliserichards do you need that?